Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <div id="header">
```

Comment:
Pssst! This looks like a missed opportunity to use the HTML5 structural tag, `<header>` instead of the semantic-less `<div id="header">`